release.sh

```diff
@@ -223,6 +214,24 @@ echo "You want to merge 'merge_master_into_develop_after_release_${MVN_VERSION_R
 echo "When you have created the PR you can press any key to continue. It's ok to merge the PR later, so wait for the Jenkins tests to finish."
 read -n 1 -s -r -p "Press any key to continue (ctrl+c to cancel)"; printf "\n\n";
 
+git checkout $MVN_VERSION_RELEASE
+
+echo "";
+echo "One-jar build takes several minutes, this is the last step and the script will complete by itself when you continue."
+read -n 1 -s -r -p "Press any key to continue to one-jar build (ctrl+c to cancel)"; printf "\n\n";
+
+# build one jar
+mvn -Passembly clean install -DskipTests
+
+echo "Starting utomated upload with sftp. Timeout is set to 1 hour!"
+
+./sftp-onejar-upload.expect $username $password $MVN_VERSION_RELEASE
```

Comment:
I also asked on the mailinglist, but just for completeness sake: what kind of tool is 'expect'?

Reply:
The sftp tool doesn’t allow you to pass a username and password. So it’s hard to automate an sftp upload. There were a number of workarounds I found when googling it, using expect was one of those options. It seems to be a fairly well known tool (and was installed by default on my Mac). It lets you automate interaction through stdin and stdout. So for sftp it going “run command”, “wait for login prompt”, “type password” and so on.